demo/test.c

```diff
@@ -2294,37 +2294,112 @@ static int test_mp_radix_size(void)
    /* number to result in a different size for every base: 67^(4 * 67) */
    mp_set(&a, 67);
    if ((err = mp_expt_u32(&a, 268u, &a)) != MP_OKAY) {
-      goto LBL_ERR;
+      goto LTM_ERR;
```

Comment:
Use LBL_ for labels now. It is more consistent like this over the whole codebase.

Reply:
Leftover from rebasing. The merging process  insisted on mangling `test_mp_radix_size` and `test_mp_div` together and it needed quite some fuddling to get them apart. Don't know why it did it.

So `LBL_` is the prefix for a **L**A**B**E**L** now?
Can I put it in the [Wiki](https://github.com/libtom/libtommath/wiki/The-Sourcecode:-Formatting-and-Style)?